docs/plugins/integrations.asciidoc

```diff
@@ -1,17 +1,13 @@
 [[plugin-integrations]]
-== Plugin integrations
+== Integration plugins
```

Comment:
What about "*Integrated* plugins."  "Integration plugins" sounds like a list of plugins that you'd use for integrations.

Reply:
we already introduced the concept in a previous article: https://www.elastic.co/blog/logstash-lines-introduce-integration-plugins.

> sounds like a list of plugins that you'd use for integrations.

It's indeed something like that, you use the integration plugins to integrate with  kafka or rabbitmq, (or jdbc/elasticsearch/aws).

Reply:
Hah..words are hard..We have used the term integrations to have umbrella plugins dedicated to integrating with a technology (which may have inputs/filters/outputs, etc). It's true that an elasticsearch output integrates with elasticsearch but the elasticsearch integration plugin will provide all that you need for this integration.

In a perfect world you'd have integration plugins that bundle inputs/filters/outputs, and all the transformation plugins like mutate, clone, kv, grok, geoip would be the non-integration plugins.

Reply:
I understand the concept, and I understand that precedent wins.  :-)
I'm OK with the way you have.